flink-clients/src/main/java/org/apache/flink/client/deployment/application/WebSubmissionJobClient.java

```diff
@@ -43,7 +43,7 @@
 
 	private final JobID jobId;
 
-	public WebSubmissionJobClient(final JobID jobId) {
+	public WebSubmissionJobClient(final JobID jobId, final ClassLoader userCodeClassloader) {
```

Comment:
The `userCodeClassloader` is not used so the parameter doesn't need to be added.

Reply:
@aljoscha Yes, I would like to remove this unused `userCodeClassloader `.